nextbike/prediction/prepare_feature.py

```diff
@@ -18,11 +20,19 @@ def create_dummies(df):
     p_bike_start = df["p_bike_start"].astype(int)
     p_spot_end = df["p_spot_end"].astype(int)
     p_bike_end = df["p_bike_end"].astype(int)
-    df.drop(["p_spot_start", "p_bike_start", "p_spot_end", "p_bike_end"], axis=1, inplace=True)
-    df_dummies = pd.concat([df, p_spot_start, p_bike_start, p_spot_end, p_bike_end], axis=1)
+    weekend = df["Weekend"].astype(int)
+    df.drop(["p_spot_start", "p_bike_start", "p_spot_end", "p_bike_end", "Weekend"], axis=1, inplace=True)
+    df_dummies = pd.concat([df, p_spot_start, p_bike_start, p_spot_end, p_bike_end, weekend], axis=1)
     return df_dummies
 
 
+def cast_datetime(pDf):
+    pDf["Start Time"] = pd.to_datetime(pDf["Start Time"], format="%Y-%m-%d %H:%M:%S").values.astype(int)
+    pDf["End Time"] = pd.to_datetime(pDf["End Time"], format="%Y-%m-%d %H:%M:%S").values.astype(int)
+
+    return pDf
```

Comment:
Die könnten wir mal in ne utils.py auslagern oder so - mache ich im datapreparation nämlich auch

Reply:
Und wozu das p bei pDf?
Wofür steht das?

Reply:
Good point!

p is for parameter. I use it as a style feature to know when I operate on a given param...

Reply:
Ah okay